docs/testplan/SmartSwitch-PMON-testplan.md

```diff
@@ -0,0 +1,939 @@
+mgmt-framework            Not OK    Service
+gnmi                      Not OK    Service
+swss:vlanmgrd             Not OK    Process
+swss:vxlanmgrd            Not OK    Process
+teamd:teammgrd            Not OK    Process
+teamd:teamsyncd           Not OK    Process
+sonic                     OK        System
+rsyslog                   OK        Process
+root-overlay              OK        Filesystem
+var-log                   OK        Filesystem
+routeCheck                OK        Program
+dualtorNeighborCheck      OK        Program
+diskCheck                 OK        Program
+vnetRouteCheck            OK        Program
+memory_check              OK        Program
+swss:buffermgrd           OK        Process
+swss:vrfmgrd              OK        Process
+swss:nbrmgrd              OK        Process
+swss:coppmgrd             OK        Process
+teamd:tlm_teamd           OK        Process
+eventd:eventd             OK        Process
+dpu-pdsagent              OK        UserDefine
+dpu-pciemgrd              OK        UserDefine
+dpu-eth_Uplink1/1_status  OK        UserDefine
+dpu-pcie_link             OK        UserDefine
+```
+#### Pass/Fail Criteria
+ *  A proper output should be delayed without any error
+
+### 1.12 Check the show system-health detail CLI on the SWITCH
+
+#### Steps
+ * Issue this CLI “show system-health detail" on the switch
+ * Verify the health is reflected properly
+ * Test all, switch, DPU0 options by issuing “show system-health detail \<options\>”
+ * Verify the health is reflected properly for the given module
+ 
+#### Verify in
+ * Switch
+   
+#### Sample Output
+```
+On Switch: “show system-health detail"
+
+System status summary
+
+  System status LED  green
+  Services:
+    Status: Not OK
+    Not Running: container_checker, lldp:lldpmgrd
+  Hardware:
+    Status: OK
+
+System services and devices monitor list
+
+Name                   Status    Type
+---------------------  --------  ----------
+container_checker      Not OK    Program
+lldp:lldpmgrd          Not OK    Process
+sonic                  OK        System
+rsyslog                OK        Process
+root-overlay           OK        Filesystem
+var-log                OK        Filesystem
+routeCheck             OK        Program
+dualtorNeighborCheck   OK        Program
+diskCheck              OK        Program
+bgp:bgpd               OK        Process
+bgp:fpmsyncd           OK        Process
+bgp:bgpcfgd            OK        Process
+PSU0.fan0              OK        Fan
+PSU1.fan0              OK        Fan
+fantray0.fan0          OK        Fan
+fantray1.fan0          OK        Fan
+fantray2.fan0          OK        Fan
+fantray3.fan0          OK        Fan
+PSU 1                  OK        PSU
+PSU 2                  OK        PSU
+
+System services and devices ignore list
+
+Name    Status    Type
+------  --------  ------
+
+On Switch: “show system-health detail DPU0"
+
+DPU0
+System status summary
+
+  System status LED  green
```

Comment:
In all the test cases if the system status is "Not OK" the LED should be red

Reply:
Fixed